packages/api/src/tx/modules/ecocredit/v1/bridge_amino.ts

```diff
@@ -0,0 +1,67 @@
+import { AminoMsg } from '@cosmjs/amino';
+import { AminoConverter } from '@cosmjs/stargate';
+import { MsgBridge } from '../../../../generated/regen/ecocredit/v1/tx';
+import { Credits } from '../../../../generated/regen/ecocredit/v1/types';
+import { AminoCredits } from './msg_cancel';
+
+const msgBridgeAminoType = 'regen.core/MsgBridge';
```

Comment:
just a heads up that we'll need to remove "core" as soon as we move to v5

Reply:
Yes we can do it in https://github.com/regen-network/regen-js/issues/62 just to keep it consistent